set.mm

```diff
@@ -748683,6 +748793,185 @@ set of all the singletons of (proper) ordered pairs over the elements of
       VN $.
   $}
 
+  ${
+    fcores.f $e |- ( ph -> F : A --> B ) $.
+    fcores.e $e |- E = ( ran F i^i dom G ) $.
+    fcores.p $e |- P = ( `' F " E ) $.
+    $( Lemma 1 for ~ fcores .  (Contributed by AV, 17-Sep-2024.) $)
+    fcoreslem1 $p |- ( ph -> P = ( `' F " dom G ) ) $=
+      ( ccnv crn cdm cin cima imaeq2i eqtri wfun wceq ffund cnvimainrn syl5eq
+      syl ) ADFKZFLGMZNZOZUDUEOZDUDEOUGJEUFUDIPQAFRUGUHSABCFHTUEFUAUCUB $.
+
+    fcores.y $e |- Y = ( F |` P ) $.
+    $( Lemma 2 for ~ fcores .  (Contributed by AV, 17-Sep-2024.) $)
+    fcoreslem2 $p |- ( ph -> ran Y = E ) $=
+      ( crn ccnv cima cres rneqi df-ima cin wceq imaeq2i 3eqtr2i wfun funimacnv
+      ffund syl wss cdm inss1 eqsstri a1i df-ss sylib eqtrd syl5eq ) AHMZFFNEOZ
+      OZEUPFDPZMFDOURHUSLQFDRDUQFKUAUBAUREFMZSZEAFUCURVATABCFIUEEFUDUFAEUTUGZVA
+      ETVBAEUTGUHZSUTJUTVCUIUJUKEUTULUMUNUO $.
+
+    $( Lemma 3 for ~ fcores .  (Contributed by AV, 13-Sep-2024.) $)
+    fcoreslem3 $p |- ( ph -> Y : P -onto-> E ) $=
+      ( wfo ccnv cdm cima cres ffnd eqid rescnvimafod fcoreslem1 reseq2d syl5eq
+      eqidd foeq123d mpbird ) ADEHMFNGOZPZEFUHQZMABUGUHEFABCFIRJUHSTADUHEEHUIAH
+      FDQUILADUHFABCDEFGIJKUAZUBUCUJAEUDUEUF $.
+
+    fcores.g $e |- ( ph -> G : C --> D ) $.
```

Comment:
From that point on, you have hypotheses with `C` and hypotheses with `dom G` (in the definition of `E`), which yields a mix which makes statements look less elegant.

Reply:
That's true, I observed this, too. I will add a TODO here and will fix this in a subsequent PR.

Reply:
Done already in this PR.